TrackletAlgorithm/ProjectionCalculator.h

```diff
@@ -37,43 +39,54 @@ PC::addProj(const TrackletProjection<TProjType> &proj, const BXType bx, Tracklet
   else
     phi >>= 3;
 
+  int ipage = trackletIndex >> 7;
```

Comment:
@aryd I think you can replace the magic 7 here with `kNBits_MemAddr`:
```suggestion
  int ipage = trackletIndex >> kNBits_MemAddr;
```

Reply:
Done.